### PR TITLE
Refactor poms into jbpm-system-test

### PR DESCRIPTION
**Pull request by gmunozfe**
Also, all components are using exclusively junit5

### CODE REVIEW

File: jbpm-system-test/jbpm-springboot-mssql-test/src/test/java/org/kie/server/springboot/samples/KieServerMSSQLTest.java [hunk 1]
```diff
@@ -49,14 +49,15 @@
 import org.springframework.test.context.DynamicPropertyRegistry;
 import org.springframework.test.context.DynamicPropertySource;
 import org.springframework.test.context.TestPropertySource;
-import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
 import org.testcontainers.containers.MSSQLServerContainer;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
```
Comment (afalhambra) [inline jbpm-system-test/jbpm-springboot-mssql-test/src/test/java/org/kie/server/springboot/samples/KieServerMSSQLTest.java:57]: these assertions can be replaced also with the junit5 assertions counterparts instead.

`org.junit.jupiter.api.Assertions`

File: jbpm-system-test/pom.xml [hunk 2]
```diff
@@ -30,6 +30,121 @@
...
+            <artifactId>junit</artifactId>
+        </exclusion>
+      </exclusions>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.testcontainers</groupId>
+      <artifactId>junit-jupiter</artifactId>
+      <version>${version.org.testcontainers}</version>
```
Comment (afalhambra) [inline jbpm-system-test/pom.xml:55]: maybe some of these dep versions could be defined in a dependencyManagement section instead, though all deps are already centralized in a single pom file, so it should be ok as is. Up to your criteria.

**Review by afalhambra — APPROVED** [review]
Nice refactoring work!

File: jbpm-system-test/jbpm-springboot-mssql-test/src/test/java/org/kie/server/springboot/samples/KieServerMSSQLTest.java (hunk 1 above)
Comment (gmunozfe) [reply jbpm-system-test/jbpm-springboot-mssql-test/src/test/java/org/kie/server/springboot/samples/KieServerMSSQLTest.java:57]: good catch @afalhambra ! thanks a lot, updated

**Comment by gmunozfe** [comment]
@mareknovotny ready to merge

**Pull request merged**